### ISSUE TITLE
Docs: Add tutorial for Svelte

### ISSUE THREAD

**Issue by AngangGuo** [IBM/carbon-components-svelte#133]
Can you please add the tutorial on how to use Carbon Components for Svelte App like the following page on Vue?

https://www.carbondesignsystem.com/tutorial/vue/overview

Thanks

**Comment by metonym** [comment]
Great idea 👍

**Comment by murunwas** [comment]
I concur with you, I find it very difficult to use this library

**Comment by dioslibre** [comment]
Check the source code in src directory. It has everything, even the storybook´s code. that's what I do, no documentation need.

**Comment by sharpcodepro** [comment]
Hi all. The Storybook is great. Basically I can copy and paste code and it saves a lot of time. The components reference/doc page is also good. However, some guides are still required especially when just getting started with Carbon. I have progressed pretty well in building a desktop app UI with Svelte/Carbon but like others, at times I am too struggling with seemingly basic stuff. Vue tutorial is great and Svelte deserves one too. Svelte/Carbon combo looks to me as a match made in heaven. Right now there is no solid UI library or framework for Svelte and Carbon can become a de-facto standard for building solid apps with Svelte.

**Comment by metonym** [comment]
@sharpcodepro I completely agree. We're working on a Carbon Svelte tutorial (similar to the React/Vue/Angular ones on the Carbon site).

**Comment by shayneoneill** [comment]
There kind of isnt any doculentation at all. The storybook for svelte seems to have React code which isnt useful, and the template repos are interesting and all but they dont really explain anything. I mean......... I cant work out how to use this thing?

**Comment by metonym** [comment]
The code for a Carbon + Svelte tutorial is located here: https://github.com/carbon-design-system/carbon-tutorial-svelte

However, it remains to be seen whether it will [make it the official docs](https://github.com/carbon-design-system/carbon-website/pull/2308).
